### ISSUE TITLE
Nothing in the window on Fedora 27

### ISSUE THREAD

**Issue by Toover** [iotaledger/wallet#967]
### Prerequisites
If you are **not** suggesting a feature, you must be able to check all of the following (place an x inside the brackets to check the box). If you cannot check all of the boxes, this issue should not be reported on GitHub as it is not a bug:

* [x] I confirm that this is an issue with the IOTA Wallet and not an exchange
* [x] I confirm that this is an issue with the wallet, not IRI
* [x] For pending transactions: I confirm that I have tried reattaching and promoting multiple times (check this box if your issue is not related to a pending transaction)
* [x] I confirm that this is not an issue related to stolen/lost funds
* [x] For private key warnings: I confirm that I am not at risk of reusing an address, or another user has verified that I am not (check this box if your issue is not related to a private key reuse)
* [x] I confirm that this issue is present on the latest version of the wallet

### Description

[Description of the bug or feature]

### Steps to Reproduce

1. Downloaded the IOTA wallet from https://github.com/iotaledger/wallet/releases/download/v2.5.6/iota-2.5.6.x86_64.rpm
2. Installed the RPM on my Fedora 27
3. Launched the app properly, chose to make a "light" wallet.

**Expected behavior:**
Shouldn't I be configuring my wallet's seed then?

**Actual behavior:**
The window is there but the background is empty (grey the first time, green the other times)

### Versions
Fedora 27 with Gnome Shell desktop (and wayland window manager, and the radeon graphics pilot)
IOTA 2.5.6

**Comment by wonkytonky** [comment]
did you try changeing node?

**Comment by Toover** [comment]
I've just tried and after having put the required neighboring nodes, I click start, and the same window (asking for the neighboring nodes) simply re-appears.

**Comment by einse** [comment]
Got same issue with my wallet just installed. Can I help you somehow?

**Comment by arigit** [comment]
+1 ...

**Comment by Woi** [comment]
Could be related to #967

**Comment by Woi** [comment]
I can confirm this issue for Fedora 27/Xfce with 2.5.6/RPM and 2.5.(3-6)/tar.gz
There was no warning when started from a shell, not even with `--verbose` and `--debug`.
[libXss.so.1](https://www.reddit.com/r/Iota/comments/6m0ze6/problem_installing_on_linux/?st=jcgb5xpe&sh=9c184d37) was installed on my system.
On Fedora26/Xfce 2.5.6/tar.gz seems to work as expected.

**Comment by arigit** [comment]
In my case - I ran into the same issue on F27/gnome/Xorg session, amdgpu open source driver.
Somewhere I read a reference that this could be related to OpenCL - then I could work around this problem enabling OpenCL from amdgpu-pro. In my install, I manually installed just the closed source AMD openCL libraries, inspired by this: https://boinc.berkeley.edu/dev/forum_thread.php?id=11417
Then: 
LD_LIBRARY_PATH=/opt/amdgpu-pro/lib64 application_name

I had done this for darktable and GIMP, and when I tried the same for this client, this client worked, the seed prompt showed up. It seems this client needs a working OpenCL install

**Comment by Woi** [comment]
@arigit Good point. Indeed my Fedora 27 system uses an ATI X1400 while my Fedora 26 is running with Intel graphics.
@all Can others facing this issue either confirm or disprove the have ATI/AMD graphics thesis?
